aidants_connect_web/templates/aidants_connect_web/usager_details.html

```diff
@@ -39,6 +39,7 @@ <h4 class="margin-bottom-0">
             </h4>
             <div class="float-right text-right">
               <a id="view_mandat_attestation" class="button" href="{% url 'mandat_visualisation' mandat_id=mandat.id %}"><span aria-hidden="true">🖨&nbsp;</span>Voir l’attestation</a>
+                <a id="renew_mandat" class="button" href="{% url 'renew_mandat' usager_id=usager.id %}"><span aria-hidden="true">🖨&nbsp;</span>Renouveller le mandat</a>
```

Comment:
Un seul L à "renouveler", sinon ça donne "renouvèler" : 
```suggestion
                <a id="renew_mandat" class="button" href="{% url 'renew_mandat' usager_id=usager.id %}"><span aria-hidden="true">🖨&nbsp;</span>Renouveler le mandat</a>
```